modules/web/client/src/main/scala/observe/ui/components/services/ObservationSyncer.scala

```diff
@@ -71,30 +67,32 @@ object ObservationSyncer:
                 .flatTap: config =>
                   props.nighttimeObservation.async.mod(_.map(_.withConfig(config)))
 
-            (fetchSummary, fetchSequence).parTupled >>= ((_, configEither) =>
-              configEither.toOption
-                .map: config =>
-                  def getBreakPoints(sequence: Option[ExecutionSequence[?]]): Set[Step.Id] =
-                    sequence
-                      .map(s => s.nextAtom +: s.possibleFuture)
-                      .orEmpty
-                      .flatMap(_.steps.toList)
-                      .collect { case s if s.breakpoint === Breakpoint.Enabled => s.id }
-                      .toSet
+            (fetchSummary, fetchSequence).parTupled.void
+            // TODO Breakpoint initialization should happen in the server, not here.
+            // Leaving the code commented here until we move it to the server.
+            // >>= ((_, configEither) =>
+            // configEither.toOption
+            //   .map: config =>
+            //     def getBreakPoints(sequence: Option[ExecutionSequence[?]]): Set[Step.Id] =
+            //       sequence
+            //         .map(s => s.nextAtom +: s.possibleFuture)
+            //         .orEmpty
+            //         .flatMap(_.steps.toList)
+            //         .collect { case s if s.breakpoint === Breakpoint.Enabled => s.id }
+            //         .toSet
 
-                  val initialBreakpoints: Set[Step.Id] =
-                    config match
-                      case InstrumentExecutionConfig.GmosNorth(executionConfig) =>
-                        getBreakPoints(executionConfig.acquisition) ++
-                          getBreakPoints(executionConfig.science)
-                      case InstrumentExecutionConfig.GmosSouth(executionConfig) =>
-                        getBreakPoints(executionConfig.acquisition) ++
-                          getBreakPoints(executionConfig.science)
+            //     val initialBreakpoints: Set[Step.Id] =
```

Comment:
Is this something that will be fixed later on?

Reply:
We should move that code to the server.